nodebuilder/p2p/p2p.go

```diff
@@ -194,24 +194,24 @@ func (m *module) PubSubPeers(_ context.Context, topic string) ([]peer.ID, error)
 //nolint:dupl
 type API struct {
 	Internal struct {
-		Info                 func(context.Context) (peer.AddrInfo, error)                         `perm:"admin"`
-		Peers                func(context.Context) ([]peer.ID, error)                             `perm:"admin"`
+		Info                 func(context.Context) (peer.AddrInfo, error)                         `perm:"public"`
+		Peers                func(context.Context) ([]peer.ID, error)                             `perm:"read"`
```

Comment:
Pretty sure that these should stay admin

Reply:
Why? it's just a set of peers that we are connected to

Reply:
ok. that makes sense. Thanks for the explanation